client/src/components/EditProfile/EditProfileTab/EditProfileTab.tsx

```diff
@@ -1,23 +1,22 @@
 import useStyles from './useStyles';
 import Typography from '@material-ui/core/Typography';
-import { FormLabel, OutlinedInput, Select, MenuItem, TextField, Button, Box, Menu, Switch } from '@material-ui/core';
-import React, { useState } from 'react';
+import { FormLabel, OutlinedInput, Select, MenuItem, TextField, Button, Box, Switch, setRef } from '@material-ui/core';
+import React, { useState, useEffect } from 'react';
 import { useFormik } from 'formik';
+import edit from '../../../helpers/APICalls/edit';
+import fetchProfile from '../../../helpers/APICalls/fetchProfile';
 
-interface UserProfile {
+interface ProfileInterface {
```

Comment:
```suggestion
interface Profile {
```
It is understood that this is an interface, we don't need to add a suffix :)